tests/test_ccs.py

```diff
@@ -16,7 +16,7 @@ def test_acquire_token_by_auth_code_flow(self):
         app = msal.ClientApplication("client_id")
         state = "foo"
         flow = app.initiate_auth_code_flow(
-            ["some", "scope"], login_hint="johndoe@contoso.com", state=state)
+            ["some", "scope"], login_hint="johndoe@contoso.com", state=state, response_mode="form_post")
```

Comment:
Most of the existing test cases were designed to test one particular behavior, and showed in the test case's file name and/or its class name.

This test case here was not meant for testing the new `response_mode` behavior. I would recommend to remove this part of change here.

You can add your dedicated test case if you prefer, but I'm even comfortable to NOT add any test case at all, given that (1) the change is minimal; (2) you have already manually tested it in your application; (3) we both reviewed this PR.

```suggestion
            ["some", "scope"], login_hint="johndoe@contoso.com", state=state)
```